src/form_builder/models.py

```diff
@@ -51,10 +51,12 @@ class Form(models.Model):
                              max_length=255,
                              help_text=_("Give your form a name."))
     slug = models.SlugField(_("Slug"),
-                            editable=False,
+                            editable=True,
                             unique=True,
                             max_length=255,
-                            blank=True)
+                            blank=True,
+                            help_text=_("This will modify the URL.")
```

Comment:
We could provide a bit more here and match the tone of other inputs. Maybe something like "Give your form a custom URL"